.design-system-version

```diff
@@ -1 +1 @@
-70.0.16
+72.2.0
```

Comment:
I think you may as well upgrade to 72.3.0

Reply:
I think we agreed in tech session that we will use 72.2.0.

Reply:
Yeah I know what you mean because it is creeping up but as this is likely to not impact this PR I thought it would be worth at least testing it. But that can be done in another PR I guess